go/tasks/pluginmachinery/coreutils/template.go

```diff
@@ -0,0 +1,55 @@
+package coreutils
```

Comment:
can we not move all files here? this looks a little off?

Reply:
I definitely want to do this, but in the past this replacement has been very expensive. With extremely large queries that may happen again, I think at-least a TODO that will measure the time it takes to replace